### ISSUE TITLE
Possible bug in the code

### ISSUE THREAD

**Issue by marcopodda** [steverab/failing-loudly#18]
Hello,
I am trying to reproduce your code.
On this line: 

https://github.com/steverab/failing-loudly/blob/42afd118237ded54c6ebef4a3417d8c1db44f76d/shift_detector.py#L73

shouldn't there be `X_val_red` instead of `X_tr_red`?

**Comment by steverab** [comment]
Hi Marco,

I agree that the variable naming here is misleading, I should change that. From a correctness point of view, though, if you trace `X_tr_red` back to where it is defined

https://github.com/steverab/failing-loudly/blob/42afd118237ded54c6ebef4a3417d8c1db44f76d/shift_detector.py#L68

you will see that the code already uses the validation set instead of the training set.

**Comment by marcopodda** [comment]
Hi Stephan,
thanks for the quick reply! I had traced it back indeed, but I was a bit confused by the naming. Glad it isn't a bug.

BTW, your paper and this repo are both awesome! :)
